content/docs/builders.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Builders"
+weight: 101
+menu: "main"
+---
+
+# Builders
+In the Getting Started tutorial, you used the `pack` CLI and the Paketo base `builder` to create a runnable image with your application. This section explains what Paketo `builders` are, and how they fit into the Paketo ecosystem.
+
+## What is a Paketo builder?
+A Paketo `builder` is an image that contains three components:
+* a set of Paketo `buildpacks`, which provide your app's dependencies,
+* a Paketo `stack`, which provides the OS layer for your app image, and
+* the [CNB lifecycle](https://buildpacks.io/docs/concepts/components/lifecycle/), which puts everything together to produce your final app image
+
+
+For more information about `builders`, visit the [buildpacks.io docs](https://buildpacks.io/docs/concepts/components/builder/). More information about `stacks` and the `CNB lifecycle` can be found there as well.
```

Comment:
This reads a little weird. I think it can be as simple as:
"For more information about builders, see [buildpacks.io](https://buildpacks.io/docs/concepts/components/builder/)" and leave out the next sentence (the stacks page will cover this)